ui/lang/sr.js

```diff
@@ -14,7 +14,9 @@ export default {
     create: 'Dodaj',
     search: 'Traži',
     filter: 'Filter',
-    refresh: 'Osveži'
+    refresh: 'Osveži',
+    expand: function (label) { return label ? `Proširi "${label}"` : 'Proširiti' },
+    collapse: function (label) { return label ? `Skupi "${label}"` : 'Kolaps' }
```

Comment:
`Kolaps` should be changed to `Skupiti`, as it is the opposite of `Proširiti` and better translation of english `Collapse` in this context. The current word will most often be translated to have non related meaning.

Reply:
Thanks for looking at this! Pushed the fix.